static/js/containers/auth/LoginPage_test.js

```diff
@@ -101,17 +103,45 @@ describe("LoginPage", () => {
 
     const { onSubmit } = inner.find("AuthEmailForm").props()
 
-    onSubmit()
+    await onSubmit()
 
     const dispatchedActions = store.getActions()
 
-    assert.lengthOf(dispatchedActions, 3)
+    assert.isAtLeast(dispatchedActions.length, 3)
     assert.equal(dispatchedActions[2].type, actions.auth.loginEmail.requestType)
     sinon.assert.calledOnce(helper.postEmailLoginStub)
-    sinon.assert.calledWith(
-      helper.postEmailLoginStub,
-      FLOW_LOGIN,
-      "test@example.com"
+    sinon.assert.calledWith(helper.postEmailLoginStub, FLOW_LOGIN, TEST_EMAIL)
+  })
+
+  it("sets some detail in the state about the user after form submission", async () => {
+    const testName = "Testuser"
+    const testImgUrl = "http://example.com/abc.jpg"
+    const { inner, store } = await renderPage()
+
+    const { onSubmit } = inner.find("AuthEmailForm").props()
+
+    helper.postEmailLoginStub.returns(
+      Promise.resolve({
+        flow:       FLOW_LOGIN,
+        state:      STATE_SUCCESS,
+        extra_data: {
```

Comment:
Take it or leave it: could simplify you code here a bit by defining `extra_data` as a `const` instead and then spreading it into your assertion at the end.